typescript/infra/scripts/middleware/portal-relayer.ts

```diff
@@ -39,16 +40,27 @@ async function relayPortalTransfers() {
 
   while (true) {
     for (const chain of Object.keys(bridgeAdapterConfigs)) {
+      console.log('chain', chain);
```

Comment:
same here, maybe we should make the message more significant, like `'Assessing chain'` or something